test/html/autocomplete.html

```diff
@@ -1,27 +1,25 @@
-<html>
-
+<!doctype html>
+<html lang="en">
 <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0, user-scalable=no">
 
     <title>Materialize - Documentation</title>
 
     <!-- Favicons -->
-    <link rel="apple-touch-icon-precomposed" href="images/favicon/apple-touch-icon-152x152.png">
+    <link rel="apple-touch-icon-precomposed" href="../../docs/images/favicon/apple-touch-icon-152x152.png">
     <meta name="msapplication-TileColor" content="#FFFFFF">
     <meta name="msapplication-TileImage" content="images/favicon/mstile-144x144.png">
-    <link rel="icon" href="../../images/favicon/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" href="../../docs/images/favicon/favicon-32x32.png" sizes="32x32">
 
     <!-- Android 5 Chrome Color -->
     <meta name="theme-color" content="#EE6E73">
 
-    <link href="../../bin/materialize.css" type="text/css" rel="stylesheet" media="screen,projection" />
-    <link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-
-    <link href="../../css/prism.css" rel="stylesheet" />
+    <link href="../../dist/css/materialize.css" type="text/css" rel="stylesheet" media="screen,projection">
+    <link href="//fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
btw, why removing "http:"?

Reply:
The "//" is short version of "https://".  I tend to use the short version these days.

Reply:
Uh sorry for the late reply...

Anyway, the `//` is relative to the current protocol that used. It will be equal to `https://` if the page is opened on https.

This will broke if you open the HTML directly without any web server. Because it tried to access `file://fonts.googleapis.com`, and it's not found in the local filesystem

Reply:
Alright. I changed the protocol.